.github/workflows/build.yml

```diff
@@ -115,174 +102,175 @@ jobs:
-        run: |
-          cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} -G Ninja
+      ##
+      # BUILD
+      ##
 
       - name: Build
         if: runner.os != 'Windows'
         run: |
           cmake --build ${{ env.BUILD_DIR }}
 
-      - name: Build on Windows
+      - name: Build (Windows)
         if: runner.os == 'Windows'
         shell: msys2 {0}
         run: |
           cmake --build ${{ env.BUILD_DIR }}
 
-      - name: Install
-        if: runner.os != 'Linux' && runner.os != 'Windows'
+      ##
+      # PACKAGE BUILDS
+      ##
+
+      - name: Package (macOS)
+        if: runner.os == 'macOS'
         run: |
           cmake --install ${{ env.BUILD_DIR }}
 
-      - name: Install on Windows
+          cd ${{ env.INSTALL_DIR }}
+          macdeployqt "PolyMC.app" -executable="PolyMC.app/Contents/MacOS/polymc" -always-overwrite
+          chmod +x "PolyMC.app/Contents/MacOS/polymc"
+          tar -czf ../PolyMC.tar.gz *
+
+      - name: Package (Windows)
         if: runner.os == 'Windows'
         shell: msys2 {0}
         run: |
           cmake --install ${{ env.BUILD_DIR }}
 
-      - name: Install on Linux
-        if: runner.os == 'Linux' && matrix.portable != true
+          cd ${{ env.INSTALL_DIR }}
+          if [ "${{ matrix.msystem }}" == "mingw32" ]; then
+            cp /mingw32/bin/libcrypto-1_1.dll /mingw32/bin/libssl-1_1.dll ./
+          elif [ "${{ matrix.msystem }}" == "mingw64" ]; then
+            cp /mingw64/bin/libcrypto-1_1-x64.dll /mingw64/bin/libssl-1_1-x64.dll ./
+          fi
+
+      - name: Package (Windows, portable)
+        if: runner.os == 'Windows'
+        shell: msys2 {0}
+        run: |
+          cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }}
+          cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }} --component portable
+
+          cd ${{ env.INSTALL_PORTABLE_DIR }}
+          if [ "${{ matrix.msystem }}" == "mingw32" ]; then
+            cp /mingw32/bin/libcrypto-1_1.dll /mingw32/bin/libssl-1_1.dll ./
+          elif [ "${{ matrix.msystem }}" == "mingw64" ]; then
+            cp /mingw64/bin/libcrypto-1_1-x64.dll /mingw64/bin/libssl-1_1-x64.dll ./
+          fi
+
+      - name: Package (Linux)
+        if: runner.os == 'Linux'
         run: |
-          DESTDIR=${{ env.INSTALL_DIR }} cmake --install ${{ env.BUILD_DIR }}
+          cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_DIR }}
+
+          cd ${{ env.INSTALL_DIR }}
+          tar -czf ../PolyMC.tar.gz *
 
-      - name: Install on Linux portable
-        if: runner.os == 'Linux' && matrix.portable == true
+      - name: Package (Linux, portable)
+        if: runner.os == 'Linux'
         run: |
-          cmake --install ${{ env.BUILD_DIR }}
+          cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }} --component portable
```

Comment:
same thing here, but you've done an oopsie and it just contains the portable.txt

Reply:
![immagine](https://user-images.githubusercontent.com/83089242/163231877-08dbb7a9-923d-4fcf-b361-df9efde9a999.png)

Reply:
lol